independent-projects/tools/common/src/main/java/io/quarkus/cli/commands/ListExtensionsCommandHandler.java

```diff
@@ -66,30 +66,41 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                     currentFormatter = this::conciseFormatter;
             }
 
-            final BuildFile buildFile = invocation.getBuildFile();
+            final BuildFile buildFile = getBuildFileSafe(invocation);
             loadedExtensions.forEach(extension -> display(extension, installed, all, currentFormatter, buildFile));
 
-            if ("concise".equalsIgnoreCase(format)) {
+            if (buildFile != null) {
+                if ("concise".equalsIgnoreCase(format)) {
+                    if (buildFile instanceof GradleBuildFile) {
+                        System.out.println("\nTo get more information, append --format=full to your command line.");
+                    } else {
+                        System.out
```

Comment:
Can it be on 2 lines with nicer formatting?

Reply:
Like this?

Reply:
👍